packages/fxa-content-server/tests/intern.js

```diff
@@ -0,0 +1,162 @@
+const fxaUntrustedOauthApp =
+  args.fxaUntrustedOauthApp || 'http://localhost:10139/';
+const fxaPaymentsRoot = args.fxaPaymentsRoot || 'http://localhost:3031/';
+const output = args.output || 'test-results.xml';
+const fxaSettingsV2Root = args.fxaSettingsV2Root || `${fxaContentRoot}settings`;
+
+// "fxaProduction" is a little overloaded in how it is used in the tests.
+// Sometimes it means real "stage" or real production configuration, but
+// sometimes it also means fxa-dev style boxes like "latest". Configuration
+// parameter "fxaDevBox" can be used as a crude way to distinguish between
+// two.
+const fxaProduction = !!args.fxaProduction;
+const fxaDevBox = !!args.fxaDevBox;
+
+const fxaToken = args.fxaToken || 'http://';
+const asyncTimeout = parseInt(args.asyncTimeout || 10000, 10);
+
+// On Circle, we bail after the first failure.
+// args.bailAfterFirstFailure comes in as a string.
+const bailAfterFirstFailure = args.bailAfterFirstFailure === 'true';
+
+const testProductId = args.testProductId || 'prod_GqM9ToKK62qjkK';
+const testPlanId = args.testPlanId || 'plan_GqM9N6qyhvxaVk';
+
+// Intern specific options are here: https://theintern.io/docs.html#Intern/4/docs/docs%2Fconfiguration.md/properties
+const config = {
+  asyncTimeout: asyncTimeout,
+  bail: bailAfterFirstFailure,
+  defaultTimeout: 45000, // 30 seconds just isn't long enough for some tests.
+  environments: {
+    browserName: 'firefox',
+    fixSessionCapabilities: 'no-detect',
+    usesHandleParameter: true,
+  },
+  filterErrorStack: true,
+
+  fxaAuthRoot: fxaAuthRoot,
+  fxaContentRoot: fxaContentRoot,
+  fxaSettingsV2Root: fxaSettingsV2Root,
+  fxaDevBox: fxaDevBox,
+  fxaEmailRoot: fxaEmailRoot,
+  fxaOAuthApp: fxaOAuthApp,
+  fxaOAuthRoot: fxaOAuthRoot,
+  fxaProduction: fxaProduction,
+  fxaProfileRoot: fxaProfileRoot,
+  fxaToken: fxaToken,
+  fxaTokenRoot: fxaTokenRoot,
+  fxaUntrustedOauthApp: fxaUntrustedOauthApp,
+  fxaPaymentsRoot,
+
+  pageLoadTimeout: 30000,
+  reporters: [
+    {
+      name: 'junit',
+      options: {
+        filename: output,
+      },
+    },
+    'runner',
+  ],
+  serverPort: 9091,
+  serverUrl: 'http://localhost:9091',
+  socketPort: 9077,
+  tunnelOptions: {
+    drivers: [
+      {
+        name: 'firefox',
+      },
+    ],
+  },
+
+  testProductId,
+  testPlanId,
+};
+
+if (args.grep) {
+  config.grep = new RegExp(args.grep, 'i');
+}
+
+if (args.suites) {
+  switch (args.suites) {
+    case 'server':
+    case 'server-resources':
+      config.functionalSuites = [];
+      config.node = {
+        suites: testsServer,
+      };
+      config.tunnelOptions = {};
+      config.environments = {
+        browserName: 'node',
+      };
+      config.reporters = 'pretty';
+      if (args.suites === 'server-resources') {
+        config.node.suites = testsServerResources;
+      }
+      break;
+  }
+}
+
+if (args.unit) {
+  config.functionalSuites.unshift('tests/functional/mocha.js');
+}
```

Comment:
Since 'tests/functional/mocha.js' no longer exists, there's no need for this if statement.